README.md

```diff
@@ -100,6 +100,12 @@ erldns_zone_cache:put_zone({
   ]}).
 ```
 
+### AXFR Support
+
+AXFR zone transfers are not currently implemented. The current "implementation" is just a stub.
+
+In order to support it, it might be needed to use a custom record handler that will for example "match" every record when the QTYPE is AXFR (252).
```

Comment:
I don't think you need to consider how it might be implemented here. The first line is sufficient.

Reply:
Addressed at https://github.com/dnsimple/erldns/pull/178/commits/5f70d02b242f414ae453e9b1e725c73198111dc5